app/app.go

```diff
@@ -644,7 +644,7 @@ func New(
 		app.GetSubspace(packetforwardtypes.ModuleName),
 		app.TransferKeeper,
 		app.IBCKeeper.ChannelKeeper,
-		app.DistrKeeper,
+		&app.StreamerKeeper,
```

Comment:
why is the distKeeper being replaced by the streamerKeeper?

Reply:
The PFM sends fee, if defined, to the community pool
The streamer also implements this interface

Reply:
I see. and why would we want it to go to the streamer and not to the community pool first and then it will be decided what to do with the funds?

Reply:
well it's the purpose of this PR
(https://github.com/dymensionxyz/dymension/issues/406)